### ISSUE TITLE
APO registration form should not allow addition of non-sdr-stemmed workgroups to roles

### ISSUE THREAD

**Issue by blalbrit** [sul-dlss/argo#1108]
Shib work disallows this - we should prevent users from adding.

**Comment by jcoyne** [comment]
I'm wondering if we should redesign the UI then?  Is there any sense in providing/showing a stem if they all must be `sdr:`?

**Comment by blalbrit** [comment]
Agreed - that seems like an appropriate approach, esp. if we think the single-stem approach is something we're locked into for the next 3-5 years.

**Comment by jcoyne** [comment]
I'm also wondering, should we change or remove this copy:
```
Common workgroups:  dlss:dpg-staff  dlss:dpg-hourlies  dlss:forensics-staff  dlss:smpl-staff  sulair:map-staff  sulair:map-hourlies
```

**Comment by blalbrit** [comment]
Yes - definitely. I don't think we need to prompt a user with a list like that anymore.
